e2e/node_similarity_test/test_two_nodes_jaccard_p/test.yml

```diff
@@ -1,21 +1,15 @@
 query: >
-        MATCH (m {id: 0}), (n {id: 1})
+        MATCH (m {id: 0})
+        WITH collect(m) as src_nodes
+        MATCH (n {id: 1})
+        WITH collect(n) as dst_nodes, src_nodes
         
-        CALL node_similarity.jaccard(m, n, "pairwise") YIELD 
-        node1 AS node1_jaccard_p, 
-        node2 AS node2_jaccard_p, 
-        similarity AS jaccard_similarity_p
-      
-        RETURN  
+        CALL node_similarity.jaccard_pairwise(src_nodes, dst_nodes) YIELD *
         
-        node1_jaccard_p.id as node1_jaccard_p, 
-        node2_jaccard_p.id as node2_jaccard_p, 
-        jaccard_similarity_p
-
+        RETURN node1.id as node1, node2.id as node2, similarity
 
 output: 
-    - node1_jaccard_p: 0
-      node2_jaccard_p: 1
-      jaccard_similarity_p: 0.5
-      
-      
+    - node1: 0
+      node2: 1
+      similarity: 0.5
+      
```

Comment:
I'm not sure how you have here this minus sign but pls fix.

Reply:
Aren’t the `-` signs just YAML formatting?

Reply:
@Josipmrden yes this i yaml formatting all tests have it?

Reply:
Oh okay, my apologies